tests/query/shoehorn.php

```diff
@@ -190,9 +199,10 @@ function test_wp_query_paged_and_posts_per_page() {
 	}
 
 	/**
-	 * @ticket 18897
+	 * @ticket https://core.trac.wordpress.org/ticket/18897
```

Comment:
`@ticket` is not actually a _valid_ phpDoc.

```suggestion
	 * @link https://core.trac.wordpress.org/ticket/18897
```

Reply:
[`@ticket` is a special annotation that the core phpunit test suite adds](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#annotations). If I recall correctly (and I may not, it's been a long time!), based on the status of the ticket, this test would either be run or skipped automatically. Was that feature not working on GitHub Actions, or did the test start failing after the annotation was changed to the full url? Or does core's test suite work differently now?

Reply:
It wasn't working to skip the tests in Github Actions or locally. I can investigate.

Reply:
Apparently that feature was removed [back in 2021](https://github.com/WordPress/wordpress-develop/commit/2985e902d4f117ad9162fdc8317644787376295a). Womp womp.

`@ticket` is still a valid annotation, but since it no longer functions in a useful way for us, I agree that `@link` is better. If we want, we can still [conditionally skip the test based on the status of the ticket](https://github.com/WordPress/wordpress-develop/blob/e2b0bfdc06b2f652ff7b3865b496325b87521d26/tests/phpunit/includes/abstract-testcase.php#L1316) by calling:

```php
$this->knownWPBug( 18897 );
```

That _might_ be better than always skipping it? 🤷

Reply:
@mboynes TIL. Thank you for the link. 〽️

Reply:
Yes, you would think that would be better, but not at the moment, as it failed due to no access to trac!!  Therefore, I reverted back...

![trac-inaccessible](https://github.com/user-attachments/assets/0fc70e6f-1a30-46b4-b8bb-70f7b74474c1)